Specs/NumberWithUnit/Korean/AgeModel.json

```diff
@@ -1,7 +1,8 @@
 [
   {
     "Input": "그녀가 다섯 살 때, 그녀는 자전거 타는 것을 배웠다.",
-    "NotSupportedByDesign": "dotNet,javascript,python,java,",
+    "NotSupportedByDesign": "javascript,python,java",
+    "IgnoreResolution": "true",
```

Comment:
For such cases, can you add a "Comment" attribute with the reason? For example, if it's a number failure or some other case?

Reply:
comments added.